bundles/org.openhab.binding.airgradient/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,202 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="airgradient"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xmlns:thing="https://openhab.org/schemas/thing-description/v1.0.0"
+	xsi:schemaLocation="https://openhab.org/schemas/thing-description/v1.0.0 https://openhab.org/schemas/thing-description-1.0.0.xsd">
+
+	<!-- A connection to the Cloud API which can have several locations (sensors) connected -->
+	<bridge-type id="airgradient-api">
+		<label>AirGradient API</label>
+		<description>Connection to the AirGradient Cloud API</description>
+
+		<representation-property>token</representation-property>
+		<config-description>
+			<parameter name="token" type="text" required="false">
+				<context>password</context>
+				<label>Token</label>
+				<description>Token to access the device</description>
+			</parameter>
+			<parameter name="hostname" type="text">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<default>https://api.airgradient.com/</default>
+				<description>Hostname or IP address of the API</description>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" unit="s" min="1">
+				<label>Refresh Interval</label>
+				<description>Interval the device is polled in sec.</description>
+				<default>600</default>
+				<advanced>true</advanced>
+			</parameter>
+		</config-description>
+	</bridge-type>
+
+	<!-- A sensor you communicate directly to over the local network -->
+	<thing-type id="airgradient-local">
+		<label>AirGradient Local Sensor</label>
+		<description>Direct network connection to a local AirGradient Sensor</description>
+
+		<channels>
+			<channel id="pm01" typeId="pm1"/>
+			<channel id="pm02" typeId="pm2"/>
+			<channel id="pm10" typeId="pm10"/>
+			<channel id="pm003-count" typeId="particle-count"/>
+			<channel id="atmp" typeId="system.outdoor-temperature"/>
+			<channel id="rhum" typeId="system.atmospheric-humidity"/>
+			<channel id="wifi" typeId="wifi"/>
+			<channel id="rco2" typeId="co2"/>
+			<channel id="tvoc" typeId="tvoc"/>
+			<channel id="leds" typeId="leds-mode"/>
+			<channel id="calibration" typeId="calibration"/>
+			<channel id="uploads-since-boot" typeId="uploads-since-boot"/>
+		</channels>
+
+		<properties>
+			<property name="name"/>
+			<property name="firmwareVersion"/>
+			<property name="serialNumber"/>
+			<property name="modelId"/>
+		</properties>
+		<representation-property>serialNo</representation-property>
```

Comment:
```suggestion
		<representation-property>serialNo</representation-property>
```
Does `serialNo` equals the `serialNumber` property? isn't there something better to use as representaiton-property?
https://www.openhab.org/docs/developer/bindings/thing-xml.html#representation-property

Reply:
Yeah, thats a bug.

Hostname is the alternative, but then it wont wont in the case of DHCP and using dns names.

The serial number of these sensors are basically the ethernet  Mac address